### PR TITLE
feat(aapd-865): add to-do tab to lpa user dashboard

### PR DESCRIPTION
**Pull request by topickering**
## Ticket Number

AAPD-865

https://pins-ds.atlassian.net/browse/AAPD-

## Description of change

Adds new To Do tab to manage my appeals page for lpa user.  also covers tickets aapd-872 - 875, covering the table and appeal number, address and appeal type columns.

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [X] My commit history in this PR is linear
- [X] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/forms-web-app/src/lib/dashboard-functions.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const extractAppealNumber = (caseReference) => {
+	return caseReference.split('/').pop();
```
Comment (beejjacobs) [inline packages/forms-web-app/src/lib/dashboard-functions.js:2]: We won't need this going forward - the caseReference will be just a number.

File: packages/forms-web-app/src/lib/dashboard-functions.js
Comment (beejjacobs) [inline packages/forms-web-app/src/lib/dashboard-functions.js:1]: Could we get some jsdoc + types on these functions? Appreciate we may not have the full data model, but hopefully enough to specify the types needed for these.

**Review by beejjacobs — COMMENTED** [review]
Looks good generally - a few minor points. OK if you want to pick these up in future tickets/as you refactor.

File: packages/forms-web-app/src/lib/dashboard-functions.js (hunk 1 above)
Comment (topickering) [reply packages/forms-web-app/src/lib/dashboard-functions.js:13]: I will leave this in for now but will refactor it away in future - it should still work for now with a number

**Review by beejjacobs — APPROVED** [review]
LGTM!

**Review by lauren-PINS — APPROVED** [review]

**Pull request merged**